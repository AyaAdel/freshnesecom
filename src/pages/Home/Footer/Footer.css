.footer {
  padding: 64px 45px;
}

.footer__links {
  display: flex;
  justify-content: space-between;
  margin-bottom: 32px;
}

.footer__copyright {
  font-size: 14px;
  line-height: 19px;
  color: #151515;
  margin: 0;
}
