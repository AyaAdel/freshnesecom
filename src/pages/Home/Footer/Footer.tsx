import React from 'react';
import { footerLinks } from 'data/footerLinks';
import ListGroup from 'components/ListGroup/ListGroup';

import './Footer.css';

export default function Footer() {
  return (
    <div className="footer">
      <div className="footer__links">
        {footerLinks.map((group, index) => (
          <ListGroup
            key={`footerlink${index}`}
            title={group.title}
            links={group.list}
          />
        ))}
      </div>
      <p className="footer__copyright">Copyright Â© 2020 petrbilek.com</p>
    </div>
  );
}
