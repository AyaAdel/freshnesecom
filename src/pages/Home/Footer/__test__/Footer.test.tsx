import React from 'react';
import renderer from 'react-test-renderer';
import { RenderComponent } from 'utils/test/render';
import Footer from '../Footer';

function MyComponent() {
  return (
    <RenderComponent>
      <Footer />
    </RenderComponent>
  );
}

it('Render Footer Correctly', () => {
  const tree = renderer.create(<MyComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
