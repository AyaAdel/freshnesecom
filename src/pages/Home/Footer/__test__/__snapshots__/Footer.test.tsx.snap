// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer Correctly 1`] = `
<div
  className="footer"
>
  <div
    className="footer__links"
  >
    <div
      className="list-group"
    >
      <h4
        className="list-group__title"
      >
        Get in touch
      </h4>
      <div
        className="list-group__links"
      >
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            About Us
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Careers
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Press Releases
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Blog
          </a>
        </div>
      </div>
    </div>
    <div
      className="list-group"
    >
      <h4
        className="list-group__title"
      >
        Connections
      </h4>
      <div
        className="list-group__links"
      >
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Facebook
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Twitter
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Instagram
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Youtube
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            LinkedIn
          </a>
        </div>
      </div>
    </div>
    <div
      className="list-group"
    >
      <h4
        className="list-group__title"
      >
        Earnings
      </h4>
      <div
        className="list-group__links"
      >
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Become an Affiliate
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Advertise your product
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Sell on Market
          </a>
        </div>
      </div>
    </div>
    <div
      className="list-group"
    >
      <h4
        className="list-group__title"
      >
        Account
      </h4>
      <div
        className="list-group__links"
      >
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Your account
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Returns Centre
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            100 % purchase protection
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Chat with us
          </a>
        </div>
        <div>
          <a
            className="list-group__link"
            href="#"
          >
            Help
          </a>
        </div>
      </div>
    </div>
  </div>
  <p
    className="footer__copyright"
  >
    Copyright Â© 2020 petrbilek.com
  </p>
</div>
`;
